test/Cshrix.FSharp.Tests/Cshrix.FSharp.Tests.fsproj

```diff
@@ -1,19 +1,23 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFramework>netcoreapp2.2</TargetFramework>
+        <TargetFramework>net5.0</TargetFramework>
+        <LangVersion>latest</LangVersion>
 
         <IsPackable>false</IsPackable>
+
+        <OutputType>WinExe</OutputType>
```

Comment:
Is `WinExe` needed here? Seems that would produce a non-portable version.

Reply:
I will change this back to classlib, I guess - I was having a lot of trouble getting your F# stuff to build at all.